.circleci/config.yml

```diff
@@ -488,8 +488,8 @@ workflows:
           extras: '[redshift-binary,itest]'
           python_version: "3.6"
           db_name: demo-itest
-          requires:
-            - test-3.8
+#          requires:  # TODO restore
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker